src/core/window.c

```diff
@@ -3597,11 +3597,6 @@ meta_window_move_resize_internal (MetaWindow          *window,
                   window->frame->rect.width,
                   window->frame->rect.height);
     }
-  else
-    {
-      frame_size_dx = 0;
```

Comment:
How about reducing scope by moving `frame_size_dx` and `frame_size_dy` into if branch if they are used only there?

Reply:
great idea!

Reply:
done